ftdetect/jira.vim

```diff
@@ -0,0 +1 @@
+autocmd BufRead,BufNewFile,BufFilePost textern-.*jira.*.* set filetype=jira
```

Comment:
Is the double `.*.*` at the end of the pattern there for a purpose, or is that an omission?